index.html

```diff
@@ -420,6 +398,243 @@ <h2>
+          reading the <a>document</a>'s <a>current orientation angle</a> via
+          <code>screen.orientation.angle</code> and listening to the
+          <code>"change"</code> event can help the developer to compensate the
+          screen orientation angle.
+        </p>
+      </section>
+      <section class='informative'>
+        <h2>
+          Interaction with Web Application Manifest
+        </h2>
+        <p>
+          The Web Application Manifest specification [[appmanifest]] allows web
+          applications to set the <a>document</a>'s <a>default orientation</a>.
+        </p>
+      </section>
+      <section class='informative'>
+        <h2>
+          Interaction with CSS Device Adaptation
+        </h2>
+        <p>
+          The CSS Device Adaptation specification [[CSS-ADAPTATION]] defines,
+          independently of this document, a way to lock the screen orientation
+          for a web page using CSS.
+        </p>
+      </section>
+      <section class='informative'>
+        <h2>
+          Interaction with Web Content Accessibility Guidelines
+        </h2>
+        <p>
+          The Web Content Accessibility Guidelines 2.1 specification [[WCAG21]]
+          includes a Success Criterion (<a href=
+          "https://www.w3.org/TR/WCAG21/#orientation">SC 1.3.4</a>) related to
+          screen orientation.
+        </p>
+        <p>
+          ​The intent of this Success Criterion is to ensure that all <a href=
+          "https://www.w3.org/TR/WCAG21/#dfn-essential">essential</a> content
+          and functionality is available regardless of the display orientation
+          (portrait or landscape). Some websites and applications automatically
+          set the screen to a particular display orientation and expect that
+          users will respond by rotating their device to match.
+        </p>
+        <p>
+          However, some users may have their devices mounted in a fixed
+          orientation (e.g. on the arm of a power wheelchair). Therefore,
+          websites and applications need to support both orientations by making
+          sure <a href=
+          "https://www.w3.org/TR/WCAG21/#dfn-essential">essential</a> content
+          and functionality is available in each orientation. While the order
+          of content and method of functionality may have differences the
+          content and functionality must always be available. When a particular
+          orientation is <a href=
+          "https://www.w3.org/TR/WCAG21/#dfn-essential">essential</a>, the user
+          needs to be advised of the orientation requirements.​
+        </p>
+      </section>
+    </section>
+    <section class='informative'>
+      <h2>
+        Examples
+      </h2>
+      <p>
+        This example shows the current screen orientation to the console every
+        time the screen orientation state changes.
+      </p>
+      <pre class='example highlight'>
```

Comment:
As a few version ago, ReSpec automatically highlights... we can hint that it's HTML tho. 

```suggestion
      <pre class='example html'>
```